internal/integration/cockroach_test.go

```diff
@@ -290,18 +290,16 @@ func TestCockroach_Enums(t *testing.T) {
 func TestCockroach_Ent(t *testing.T) {
 	crdbRun(t, func(t *crdbTest) {
 		// Cockroach doesn't support macaddr but its in the integration tests of ent
-		macClmn := entmigrate.FieldTypesColumns[0]
-		var schemaType map[string]string
+		var st map[string]string
 		for _, ff := range entmigrate.FieldTypesColumns {
-			if ff.Name == "mac" {
-				macClmn = ff
-				schemaType = ff.SchemaType
-				ff.SchemaType = nil
+			if st = ff.SchemaType; st[dialect.Postgres] == "macaddr" {
+				c := ff
+				t.Cleanup(func() {
+					c.SchemaType = st
+				})
+				c.SchemaType = nil
 			}
 		}
```

Comment:
there can be more than one "macaddr" column at some point. Use if-level variable instead.